reference/functions/packageupdatesmatching.markdown

```diff
@@ -34,12 +34,23 @@ The return is a data container with a list of package descriptions, looking like
 * `arch_regex` - Regular expression matching package architecutre
 * `method_regex` - Regular expression matching package method (apt-get, rpm, etc ...)
 
+**IMPORTANT:** Please note that you need to provide ```package_inventory``` attribute in ```body common control``` in order to be able to use this function. Also depending on the value(s) of ```package_inventory``` only packages from selected package modules will be returned. For more information about ```package_inventory``` please read [`package_inventory`][Components and Common Control#package_inventory] section.
+
+
 **Example:**
 
 ```cf3
       "all_package_updates" data => packageupdatesmatching(".*", ".*", ".*", ".*");
 ```
 
+**Refresh rules:**
```

Comment:
Maybe also comment that network is only hit if the ifelapsed has expired, regardless of the rules below.